mapstruct-lombok/build.gradle

```diff
@@ -1,6 +1,5 @@
 plugins {
     id 'java'
-    id 'net.ltgt.apt' version '0.9'
```

Comment:
I was going to suggest you to do that (Delegate the IDE build / run actions to Gradle). That way everything works as from the commant line.